timrlink.net.CLI/ProjectTimeEntry.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+
+namespace timrlink.net.CLI
+{
+    internal class ProjectTimeEntry
```

Comment:
what do we need this new type for now?

Reply:
Thought this will come up. 😅🙈

Reply:
Cause I cherry picked from my other changes.